src/pages/[...slug].tsx

```diff
@@ -1,110 +1,75 @@
-import {
-  GetStaticPathsContext,
-  GetStaticPathsResult,
-  GetStaticPropsContext,
-  GetStaticPropsResult,
-} from 'next'
-import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
-import { drupalClient } from '@/utils/drupalClient'
-import { entityMeta } from '@/lib/delegators/entityMetaProvider'
-import { NodeTypes } from '@/types/node'
-import { getGlobalElements } from '@/lib/context/getGlobalElements'
-import Layout from '@/components/layout'
-import { generalEntityDataService } from '@/lib/delegators/generalEntityDataService'
+import * as React from "react"
+import { GetStaticPathsResult } from "next"
 
-interface EntityProps {
-  entityProps: any
+import { drupalClient } from "@/utils/drupalClient"
+import { queries } from "@/data/queries"
+import { NewsStoryPageProps } from "../types"
```

Comment:
We load this here to type the resource that gets returned from our `getStaticProps` function. In practice this would be a union type.